ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeOwnersReview.vue

```diff
@@ -6,22 +6,25 @@
     </header>
 
     <div :class="{ 'border-error-left': !getStepValidation(MhrSectVal.HOME_OWNERS_VALID) }">
-      <div v-show="!getStepValidation(MhrSectVal.HOME_OWNERS_VALID)" class="px-6 py-8">
-        <v-icon color="error">mdi-information-outline</v-icon>
-        <span class="error-text mx-1">This step is unfinished.</span>
-        <router-link :to="{ path: `/${RouteNames.MHR_REGISTRATION}/${RouteNames.HOME_OWNERS}` }"
-          >Return to this step to complete it.
-        </router-link>
-      </div>
-      <section class="px-6 my-2" v-if="hasHomeOwners || (hasGroups && showGroups)">
+      <section v-show="!getStepValidation(MhrSectVal.HOME_OWNERS_VALID)"
+      :class="hasHomeOwners ? 'pt-30px px-6' : 'px-6 py-8'">
+        <span>
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text mx-1">This step is unfinished.</span>
+          <router-link :to="{ path: `/${RouteNames.MHR_REGISTRATION}/${RouteNames.HOME_OWNERS}` }"
+          >Return to this step to complete it.</router-link>
+        </span>
+      </section>
```

Comment:
Could you explain the change to `section` tag vs div? Is it based on validation scroll to effect?

Also refer to this for helper classes: https://vuetifyjs.com/en/styles/spacing/#how-it-works
Is `pt-30px` from vuetify or custom? 

Also, need an indent on the class binding `:class`

Reply:
Switched to section simply for consistency with the other section tags within the div. Didn't see any visual differences, is one preferred over the other in this case?

Thanks for the helper class link, pt-30px is a custom class in base.scss

Good catch